metrics/repl.lag/lag.go

```diff
@@ -198,3 +265,58 @@ func (c *Lag) Collect(ctx context.Context, levelName string) ([]blip.MetricValue
 	}
 	return []blip.MetricValue{m}, nil
 }
+
+func (c *Lag) collectLagFromPFS(ctx context.Context, levelName string) ([]blip.MetricValue, error) {
+	// if isReplCheck is supplied, check if it's a replica
+	defaultLag := func() ([]blip.MetricValue, error) {
```

Comment:
Is there a reason this needs to be an anonymous function definition? Can you just break it into a standard function since it seems like you're just justing the same default lag value? It also doesn't ever return an error so the signature is a bit misleading.

Reply:
You are right, it can just be a value. Doesn't need to be a func, I was probably anticipating it doing more when I wrote the func, but I don't need it to be a func.